src/Cdn/Cdn/help/Disable-AzCdnCustomDomainHttps.md

```diff
@@ -0,0 +1,217 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Cdn.dll-Help.xml
+Module Name: Az.Cdn
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cdn/disable-azcdncustomdomainhttps
+schema: 2.0.0
+---
+
+# Disable-AzCdnCustomDomainHttps
+
+## SYNOPSIS
+Disables Custom Domain HTTPS.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Disable-AzCdnCustomDomainHttps -CustomDomainName <String> -EndpointName <String> -ProfileName <String>
```

Comment:
For every cmdlets, the order of parameter should be: identity from top level down, all other mandatory parameters, then all non-mandatory parameters. The way to fix the order here is to order the parameters in cs files in the correct order. Please fix everywhere and regen the help files.

Reply:
Fixed